src/MIDebugEngine/MIDebugEngine.csproj

```diff
@@ -141,6 +141,7 @@
     <Compile Include="Engine.Impl\EngineCallback.cs" />
     <Compile Include="Engine.Impl\EngineUtils.cs" />
     <Compile Include="Engine.Impl\ExceptionManager.cs" />
+    <Compile Include="AD7.Impl\HashAlgorithmId.cs" />
```

Comment:
VS organizes the entries in the `ItemGroup` alphabetically.

So the next time this csproj file is edited by VS, it will create an edit moving this line to its "correct" position (as of now, after the line for `AD7.Impl\EngineConstants.cs`).